packages/x-tree-view/src/useTreeItem2/useTreeItem2.ts

```diff
@@ -38,10 +40,18 @@ export const useTreeItem2 = <
   const { id, itemId, label, children, rootRef } = parameters;
 
   const { rootRef: pluginRootRef, contentRef } = runItemPlugins(parameters);
-  const { interactions, status } = useTreeItem2Utils({ itemId, children });
+  const { interactions, status } = useTreeItem2Utils({
+    itemId,
+    children,
+  });
   const idAttribute = instance.getTreeItemIdAttribute(itemId, id);
   const handleRootRef = useForkRef(rootRef, pluginRootRef)!;
   const checkboxRef = React.useRef<HTMLButtonElement>(null);
+  const iconContainerRef = React.useRef<HTMLDivElement>(null);
+  const inputRef = React.useRef<HTMLInputElement>(null);
+
+  const isBeingEdited = status.editing;
```

Comment:
I think we can skip this variable and directly use `status.editing` since it's as clear and the variable is only used once.